### DISCUSSION TITLE
有没有考虑过vp编解码的方案呢？

### DISCUSSION THREAD

**Discussion by WoYang** [Tencent/vap#82] [Ideas] [answered]
比如说vp6a编解码，视频是带alpha通道的，但是需要软解，性能是个问题。另外wp8、vp9都可以带alpha通道，再按vap的思路把最终的合成渲染出来，不知是否可行。

**Comment by hexleo** [comment, marked as answer]
有考虑过vp9，但如果想要硬件解码兼容性是要考虑的问题，目前也就h264兼容性最好。 软解性能太低，而且有开源协议的问题，所以我们不支持软解。你们可以自己增加软解扩展。
